### ISSUE TITLE
Add validation to the social and game channel text fields

### ISSUE THREAD

**Issue by tinashe404** [atsign-company/at_wavi_app#154]
**Is your feature request related to a problem? Please describe.**
Currently you are able to enter anything into the text fields in the social and game channel sections of the app. For example, you can enter a facebook link into the Youtube textfield under Social Channels.

**Describe the solution you'd like**
It would be good to add validation to the text fields so that for example, if the social channel is for facebook it will either only take a username or a facebook url

**Comment by everly-gif** [comment]
Hi @tinashe404 I would like to work on this, if its up for grabs.

**Comment by Pal18jain** [comment]
Hi, I am Palak Jain, a contributor to OpenForce 2022. I would like to work on this issue. I would be making a PR as soon as I am done with resolving the issue. Thank you.

**Comment by everly-gif** [comment]
Hi @Pal18jain,  Just a heads up as I am also working on this issue and I believe issues are given first come first serve.
@tinashe404

**Comment by yahu1031** [comment]
We will assign the tickets for 3 max. We take PR as first server/the best approach

**Comment by keyurgit45** [comment]
Hi @tinashe404 @yahu1031, I am Keyur Gandhi, a contributor to OpenForce 2022. I would like to work on this issue. I would be making a PR as soon as I am done with resolving the issue. Thank you

**Comment by keyurgit45** [comment]
@tinashe404 suppose my twitter account url is https://www.twitter.com/keyur45. So I should enter the url or my username (@keyur45) in the text field. Am I right?

**Comment by tinashe404** [comment]
@keyurgit45 Yes you are right. And yes, you can check if it's present as one of your forms of validation.
